addons/mail/static/src/models/rtc_call_participant_card.js

```diff
@@ -154,6 +209,25 @@ registerModel({
             default: 'Anonymous',
             compute: '_computeName',
         }),
+        /**
+         * The text describing the outbound ice connection candidate type.
+         */
+        outboundConnectionTypeText: attr({
+            compute: '_computeOutboundConnectionTypeText',
+        }),
+        /**
+         * The protocols for each RTC ICE candidate types.
+         */
+        protocolsByTypes: attr({
+            default: {
+                'host': "",
+                'srflx': "TURN",
+                'prflx': "TURN",
+                'relay': "STUN",
+            },
+            readonly: true,
+            required: true,
+        }),
```

Comment:
looks like a global variable, this shouldn't be duplicated for each participant card

Reply:
Also object in fields are not really recommended, as this often highlights it should be a relation instead and we are lacking an extra model... But I guess in this case it could be acceptable.